spec/amp-framework-hosting.md

```diff
@@ -0,0 +1,263 @@
+# Hosting the AMP framework
+
+You can host the AMP framework and components from your own server or CDN. This feature has a number of applications. For example, you can...
+
+- test and demonstrate changes to the framework or components.
+- set a release cadence that matches your development cycle.
+- deliver the AMP framework in regions where `cdn.ampproject.org` may not be available.
+- serve AMP pages and the framework from the same host, potentially improving content delivery times.
+
+The AMP Project is looking into options for [validation](https://amp.dev/documentation/guides-and-tutorials/learn/validation-workflow/validate_amp/) of AMP pages that use an AMP framework hosted outside of `cdn.ampproject.org`. As of April 2020, these AMP pages do not pass validation.
+
+## Versioning
+
+This document makes frequent use of the terms "version" and "runtime version (rtv)". These two versions are slightly different. When the AMP framework is built, it is assigned a 13-digit _version_. When the AMP framework is served, a config number prefixes the version, resulting in a 15-digit _runtime version_. The build system and runtime enforce these version formats.
+
+The AMP Project has adopted the following conventions:
+
+- Version: the commit time of the last commit in the active branch
```

Comment:
You can link to #16631 for added context

Reply:
Done